plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go

```diff
@@ -83,7 +83,8 @@ func (d *defaultServicePlan) Admit(a admission.Attributes) error {
 		if !apierrors.IsNotFound(err) {
 			return admission.NewForbidden(a, err)
 		}
-		msg := fmt.Sprintf("ClusterServiceClass %q does not exist, can not figure out the default ClusterServicePlan.", instance.Spec.ClusterServiceClassExternalName)
+		msg := fmt.Sprintf("ClusterServiceClass %c does not exist, can not figure out the default ClusterServicePlan.",
```

Comment:
I am getting an error from this line today. 
```
plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go:86: arg instance.Spec.PlanReference for printf verb %c of wrong type: github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog.PlanReference
exit status 1
```

Reply:
```
# Verify that vendor/ is in sync with Gopkg.lock
docker run --security-opt label:disable --rm -v /Users/nicholss/go/src/github.com/kubernetes-incubator/service-catalog:/go/src/github.com/kubernetes-incubator/service-catalog -v /Users/nicholss/go/src/github.com/kubernetes-incubator/service-catalog/.pkg:/go/pkg --env AZURE_STORAGE_CONNECTION_STRING scbuildimage build/verify-vendor.sh
Verified that vendor/ is in sync with Gopkg.lock
Running gofmt:
Running golint and go vet:
docker run --security-opt label:disable --rm -v /Users/nicholss/go/src/github.com/kubernetes-incubator/service-catalog:/go/src/github.com/kubernetes-incubator/service-catalog -v /Users/nicholss/go/src/github.com/kubernetes-incubator/service-catalog/.pkg:/go/pkg --env AZURE_STORAGE_CONNECTION_STRING scbuildimage go vet github.com/kubernetes-incubator/service-catalog/...
warning: ignoring symlink /go/src/github.com/kubernetes-incubator/service-catalog/docsite/docs
warning: ignoring symlink /go/src/github.com/kubernetes-incubator/service-catalog/vendor/github.com/coreos/etcd/cmd/etcd
warning: ignoring symlink /go/src/github.com/kubernetes-incubator/service-catalog/vendor/github.com/prometheus/procfs/fixtures/self
pkg/controller/controller.go:506: arg instance.Spec.PlanReference for printf verb %c of wrong type: github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1.PlanReference
pkg/controller/controller_instance.go:931: arg instance.Spec.PlanReference for printf verb %c of wrong type: github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1.PlanReference
pkg/controller/controller_instance.go:965: arg instance.Spec.PlanReference for printf verb %c of wrong type: github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1.PlanReference
pkg/controller/controller_instance.go:1041: arg instance.Spec.PlanReference for printf verb %b of wrong type: github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1.PlanReference
pkg/controller/controller_clusterid_test.go:85: github.com/kubernetes-incubator/service-catalog/vendor/k8s.io/apimachinery/pkg/runtime/schema.GroupResource composite literal uses unkeyed fields
pkg/controller/controller_clusterid_test.go:110: github.com/kubernetes-incubator/service-catalog/vendor/k8s.io/apimachinery/pkg/runtime/schema.GroupResource composite literal uses unkeyed fields
pkg/controller/controller_instance_test.go:4332: arg instance.Spec.PlanReference for printf verb %c of wrong type: github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1.PlanReference
pkg/controller/controller_test.go:556: github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1.ClusterServiceClassStatus composite literal uses unkeyed fields
exit status 1
plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go:86: arg instance.Spec.PlanReference for printf verb %c of wrong type: github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog.PlanReference
exit status 1
make: *** [verify] Error 1
```

Reply:
This was caused by a stale build image and was fixed after running `make clean-build-image`.